src/tab/Tab.ts

```diff
@@ -0,0 +1,72 @@
+import { css, html, TemplateResult } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import { OmniElement } from '../core/OmniElement.js';
+
+/**
+ * Control that can be used to display slotted content, for use within an omni-tabs component.
+ *
+ * @import
+ * ```js
+ * import '@capitec/omni-components/tab';
+ * ```
+ * @example
+ * html```
+ * <omni-tab header='Tab 1'>
+ *  <span>Tab 1 Content</span>
+ * </omni-tab>
+ * ```
+ *
+ * @element omni-tab
+ *
+ * @slot - Content to render inside the component body.
+ *
+ */
+@customElement('omni-tab')
+export class Tab extends OmniElement {
+    /**
+     * Tab header label.
+     * @attr
+     */
+    @property({ type: String, reflect: true }) header?: string;
+
+    /**
+     * Indicator if the component is disabled.
+     * @attr
+     */
+    @property({ type: Boolean, reflect: true }) disabled?: boolean;
+
+    /**
+     * Indicator if the component is active.
+     * @attr
+     */
+    @property({ type: Boolean, reflect: true }) active?: boolean;
+
+    static override get styles() {
+        return [
+            super.styles,
+            css`
+            :host {
```

Comment:
Check code indentation here. Lines 48 to 56 needs to be indented one tab.

Reply:
Updated